server/webapp/WEB-INF/rails/webpack/views/components/forms/forms.scss

```diff
@@ -113,7 +114,8 @@ input[type="checkbox"].form-control { //sass-lint:disable-line no-qualifying-ele
 }
 
 .form-input-reset {
-  margin: 5px 0 0 0;
+  margin: 5px 0 0;
+  display: inline-block;
```

Comment:
Doesn't alter its appearance/layout on the Config Repos modal (due to the width of the modal), so no breaking change here.

But it looks more intuitive on the Add Pipelines page.